usage/cli/src/main/java/brooklyn/cli/ItemLister.java

```diff
@@ -66,6 +68,9 @@
 public class ItemLister {
```

Comment:
I didn't know about this before! Needs documented somewhere, perhaps in `docs/guide/ops/launch.md` where other CLI options are documented such as  `brooklyn cloud-compute`.